torchvision/prototype/transforms/functional/_color.py

```diff
@@ -119,7 +122,29 @@ def adjust_sharpness_image_tensor(image: torch.Tensor, sharpness_factor: float)
     else:
         needs_unsquash = False
 
-    output = _blend(image, _FT._blurred_degenerate_image(image), sharpness_factor)
+    kernel_dtype = image.dtype if fp else torch.float32
```

Comment:
To ease review, here is the old implementation: https://github.com/pytorch/vision/blob/bf58902b2fd881c760cd2eeacfae2d7c468ebf1f/torchvision/transforms/functional_tensor.py#L803